README.md

```diff
@@ -271,6 +272,60 @@ processes and release the occupied TCP ports:
 killall peer
 ```
 
+### Fault Tolerance ###
+
+The above example shows a simple normal case of the consensus network.
+Our next interest is how the system behaves when some replicas are faulty.
+
+#### Crash Fault on Backup ####
+
+The simplest faulty case is crash fault on backup replicas. Note that
+we don't tolerate any type of fault on primary replica until view change
+operation is implemented.
+
+Let's restart the network, and keep in mind the process IDs of each
+replica process.
+
+```sh
+$ bin/peer run 0 &
+[1] 16899
+$ bin/peer run 1 &
+[2] 16916
+$ bin/peer run 2 &
+[3] 16923
+```
+
+Make sure that all replicas are properly working by sending a request:
+
+```sh
+$ bin/peer request First request
+2019/04/10 02:46:06 Received Reply message from replica 1
+2019/04/10 02:46:06 Received Reply message from replica 0
```

Comment:
I think, for the sake of brevity, we'd better eliminate the lines of debug output.

Reply:
OK, removed these lines.